ice40/icestorm.cmake

```diff
@@ -5,7 +5,7 @@ function(icestorm_setup)
   set(ICESTORM_SRC ${symbiflow-arch-defs_SOURCE_DIR}/third_party/icestorm CACHE PATH "Path to icestorm repository")
   set(PYUTILS_PATH ${symbiflow-arch-defs_SOURCE_DIR}/utils:${symbiflow-arch-defs_SOURCE_DIR}/ice40/utils/fasm_icebox)
 
-  get_target_property_required(PYTHON_PREFIX env CONDA_DIR)
+  get_target_property(PYTHON_PREFIX env CONDA_DIR)
```

Comment:
This needs to be fixed, just changing it to get_target_property will break things.

Reply:
Fixed.